main.js

```diff
@@ -0,0 +1,203 @@
+// Grab HTML elements
+const fromTower = document.querySelector(".fromTower");
+const toTower = document.querySelector(".toTower");
+const moveButton = document.querySelector("#moveButton");
+const reset = document.querySelector("#reset");
+
+const tower1 = document.querySelector(".tower1");
+const tower2 = document.querySelector(".tower2");
+const tower3 = document.querySelector(".tower3");
+
+const disks = [
+  document.querySelector(".disk-1"),
+  document.querySelector(".disk-2"),
+  document.querySelector(".disk-3"),
+  document.querySelector(".disk-4"),
+  document.querySelector(".disk-5"),
+];

```

Comment:
any ideas how we can do less DRY here and more efficient code?
A function that receives a discNumber can do something returning 
```suggestion
document.querySelector(`disk-${discNumber}`)
```